src/app_sensor.h

```diff
@@ -227,6 +228,23 @@ void m_sensors_init (void); //!< Give Ceedling a handle to initialize structs.
   }
 #endif
 
+#if APP_SENSOR_ENVIRONMENTAL_MCU_ENABLED
+#define APP_SENSOR_ENVIRONMENTAL_MCU_DEFAULT_CFG  \
+  {                                               \
+    .sensor = {0},                                \
+    .init = &ri_environmental_mcu_init,           \
+    .configuration = {0},                         \
+    .nvm_file = APP_FLASH_SENSOR_FILE,            \
+    .nvm_record = APP_FLASH_SENSOR_NTC_RECORD,    \
```

Comment:
APP_FLASH_SENSOR_NTC_RECORD -> APP_FLASH_SENSOR_MCU_RECORD

#define in app_config.h

Reply:
Defined as `#define APP_FLASH_SENSOR_ENVI_RECORD     (0x52U)`